src/journeys_utils.js

```diff
@@ -460,6 +469,13 @@ journeys_utils.animateBannerEntrance = function(banner, cssIframeContainer) {
 				// check if safeAreaRequired is true or not
 				if (journeys_utils.journeyLinkData && journeys_utils.journeyLinkData['journey_link_data'] && !journeys_utils.journeyLinkData['journey_link_data']['safeAreaRequired']) {
 					banner.style.bottom = '0';
+
+					if (metadata.type === "center_overlay") {
```

Comment:
Lets move "center_overlay" to const if possible

Reply:
done